backend/src/tasks/hibp.ts

```diff
@@ -87,7 +87,7 @@ export const handler = async (commandOptions: CommandOptions) => {
           state: 'open',
           source: 'hibp',
           needsPopulation: true,
-          exposedEmails: results
+          structuredData: results
```

Comment:
Let's store it in this way:

```suggestion
          structuredData: { emails: results }
```

This way, we can store additional information relating to the scan in different keys of the `structuredData` object, as necessary.